test/test_dtype.py

```diff
@@ -189,5 +189,13 @@ def test_strs(self):
     self.assertEqual(str(dtypes.imagef((1,2,4))), "dtypes.imagef((1, 2, 4))")
     self.assertEqual(str(PtrDType(dtypes.float32)), "ptr.dtypes.float")
 
+class TestHelpers(unittest.TestCase):
+  def test_helpers(self):
```

Comment:
consider adding cases for `uint` and `uint.vec` and its interaction with int and float